### PR TITLE
Added instructions to run `sos report` for EDPM compute nodes

### PR DESCRIPTION
**Pull request by rebtoor**
Also: 
- Fixed markdown errors

Depends-On: https://github.com/sosreport/sos/pull/3414
JIRA [OSPRH-5744](https://issues.redhat.com/browse/OSPRH-5744)

### CODE REVIEW

File: README.md
Comment (fmount) [inline README.md:187]: I would rather add a "Depends-On" (a logical depends on given we don't have any CI here), and wait until the patch [1] merges.
When [1] merges, we can change this statement to describe how to run a standalone `sos report` on an existing compute node (which is basically described below).

[1]  https://github.com/sosreport/sos/pull/3414

File: README.md
Comment (fmount) [inline README.md:191]: As long as [1] merges, you can git clone `https://github.com/sosreport/sos` on the main branch

[1] https://github.com/sosreport/sos/pull/3414

File: README.md
Comment (rebtoor) [reply README.md:187]: Totally agree on this.

File: README.md
Comment (rebtoor) [reply README.md:191]: I guess that we should remove the clone step, since we expect that sos will be already installed on compute nodes. WDYT?

Comment (fmount) [reply README.md:191]: +1, and we can provide instructions to run the sosreport both from a coreos node and from an edpm one using the new profile

File: README.md
Comment (Akrog) [inline README.md:78]: -1: This prefix was written on purpose to reflect that this is running within the container.

File: README.md
Comment (Akrog) [inline README.md:187]: For OCP cluster nodes the SOS report should be gathered using the mechanism that is being added in [PR#19](https://github.com/openstack-k8s-operators/openstack-must-gather/pull/19), though I think it has value to document the manual mechanism here.

File: README.md
Comment (Akrog) [inline README.md:304]: -1: This is insufficient, we also need storage related plugins such as `block,iscsi,scsi,multipath,devicemapper,devices,lvm2,nfs,cifs,nvme,udev`

File: README.md
Comment (Akrog) [inline README.md:307]: For other reviewers, [that is being added in this PR](https://github.com/openstack-k8s-operators/openstack-must-gather/pull/18)

**Review by Akrog — CHANGES_REQUESTED** [review]
Thank you for the documentation effort. :-)

File: README.md
Comment (rebtoor) [reply README.md:304]: ack!

File: README.md
Comment (Akrog) [reply README.md:307]: @fmount Should we wait until the `sos` PR gets merged before merging this one?

Comment (fmount) [reply README.md:307]: Yeah, this is the idea, we can publish the `README` after that patch merges (I'm not sure how the building/shipping process works on the `sos` project, so we might have to wait a bit more).
In general I agree to hold this one until https://github.com/sosreport/sos/pull/3414 lands.

Comment (Akrog) [reply README.md:307]: Following these instructions I have this patch in my local repo to do this upon request with must-gather: https://github.com/Akrog/openstack-must-gather/commit/c0fc8872727dcba84b68bd8ba7b0e3f65e843dd0

**Comment by rebtoor** [comment]
@fmount as you can see sos patch has been merged, but there's no new release of sos (yet). I'd wait to merge this PR until we'll have a new release of sos.

**Review by fmount — APPROVED** [review]
/lgtm

**Pull request merged**
